server/controller/questions.js

```diff
@@ -28,3 +28,28 @@ exports.post = async (request, response) => {
     response.status(500).send('Server Error');
   }
 };
+
+exports.put = async (request, response) => {
+  try {
+    const { id: loggedUserId } = request;
+    const data = request.body;
+    const { userId, question, questionId: id } = data;
+    if (loggedUserId === userId) {
+      const updateResult = await questions.update(
+        { question },
+        { where: { id } },
+      );
+      if (updateResult.length) {
+        response
+          .status(200)
+          .send('Your question has been successfully modified');
+      } else {
+        throw new Error();
```

Comment:
@HemaSAli In this case, is updateResult empty because no question matched that id?  If so, this could become a 404 response.

Reply:
Thank's tom
I think here I must return a response with 404 instead of throw error, as you said